.github/workflows/release.yml

```diff
@@ -186,6 +186,16 @@ jobs:
 
     steps:
 
+        # workaround to strip bugfix number from semver (only to make 1.1 release) I will change this in the future
+      - name: Set version env vars
+        # VERSION will be available only in the next step
+        run: |
+          echo "VERSION=${GITHUB_REF##*/}" >> $GITHUB_ENV
+
+      - name: Set installer env vars
+        run: |
+          echo INSTALLER_VARS="project.outputDirectory=$PWD project.version=${VERSION%.*} workspace=$PWD realname=Arduino_Create_Bridge" >> $GITHUB_ENV
+
```

Comment:
The purpose of this workaround is to consider only `major.minor` in the string used in the name of the installer. Without this and by using only `GITHUB_REF##*/` the result will be something like `1.1.251` instead of the desired `1.1`

The `>> $GITHUB_ENV` trick is used because in not possible to use an env variable defined inside `env` inside the same environment